how-to/how-to-template.md

```diff
@@ -0,0 +1,54 @@
+<!-- Copy this Template. -->
+<!-- Describe the title of your article by replacing "How To Template" with the page name you want to publish to. -->
+
+# Title
+
+::information_source: Before using this template, read the accompanying [how-to template guide](how-to-template-guide.md)
+
+## Overview
+
+This guide explains how to {insert a brief description of the task}.
+
+:information_source: Here are some examples:
+
+- This guide explains how to comment on a pull request.
+- This guide explains how to create or delete a branch on Github.
```

Comment:
I believe GitHub should be with a capital H.

Reply:
Fixed